package/system-role-xen.spec

```diff
@@ -36,7 +36,7 @@ BuildRequires:  yast2-installation-control >= 4.0.0
 
 Url:            https://github.com/yast/system-role-xen
 AutoReqProv:    off
-Version:        15.1.2
+Version:        15.1.3
```

Comment:
Same note about version as in https://github.com/yast/system-role-kvm/pull/18/files#r552216039

Reply:
ok. Done. Thanks for the review